orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -0,0 +1,200 @@
+package com.sap.ai.sdk.orchestration;
```

Comment:
```suggestion
package com.sap.ai.sdk.orchestration.client;
```
Why is the package not `client`?